cranelift-faerie/src/backend.rs

```diff
@@ -141,6 +143,8 @@ impl Backend for FaerieBackend {
         total_size: u32,
     ) -> ModuleResult<FaerieCompiledFunction> {
         let mut code: Vec<u8> = vec![0; total_size as usize];
+        // TODO: Replace this with FaerieStackmapSink once it is implemented.
```

Comment:
What do you see as the possible uses for this in the faerie backend?

Reply:
There are use cases for compiling WebAssembly AOT into native binaries, such as to run on small devices where compilation would be prohibitively slow. In such contexts, we'll need the same stackmap support we need for JIT use cases.

Reply:
I see, so this will just need to collect the info similar to what is done with traps?

Reply:
Yep.